packages/client/src/components/FeaturedMovie/FeaturedMovie.jsx

```diff
@@ -2,6 +2,7 @@ import PlayArrowIcon from '@mui/icons-material/PlayArrow';
 import { Box, IconButton, Typography } from '@mui/material';
 import React, { useEffect, useState } from 'react';
 import { apiURL } from '../../apiURL';
+import styled from '@emotion/styled';
```

Comment:
nice idea with using styled components! 
What I would suggest would be:
- turning all the inline styles into styled components (not to mix both)
- having reusable styled components imported from an internal library (like a directory with reusable components)

Maybe this is a part of the plan? Just checking :)

Reply:
Thank you so much for your review :) 

I have some questions that we have a lot of inline styles but some of them are using only one place in THIS component. 
Do we still need to create the styled component? 

In my code, I also define the object called centerElement that is using to add styles in 'Box'  element ( I am using this object in 2 places in  my code but because some styles need to added extra so I decided to spread the object and add extra styles  but I am not sure if I need to create styled component for this and how can I name it meaningful since its just set the element in the  center 

**having reusable styled components imported from an internal library (like a directory with reusable components**
Yes, We have talked about styled component but not sure about if we will create a file for sharing component

Reply:
@magdazelena  I have fixed all my inline styles into styled components :) 

**having reusable styled components imported from an internal library (like a directory with reusable components** 
It is for the next stage not for now

Reply:
> @Kositthai _Do we still need to create the styled component?_
> 
> This is up to you and the team to decide, but if I was in the team, I would vote yes :) I really like to keep my sections of logic separated - and the same for other parts, like styling (css), schemas(db), skeleton (html/jsx) and logic (js).
> 
> It does not have to mean that those all should be in separate files (rule of thumb is separate the reusable), but it makes it easier if they are separated in some way.
> 
> So actually answering your question: Yes, I would remove all inline styles and make all components that have styles styled components. You made and excellent choice in separating `centerElement` and then reusing it and you can by all means keep it, even if you decide to make all your components styled. Always a good pattern not to repeat logic :)
> 
> As for naming goes: use it for your advantage! You are welcome to call your components `TopRowBox` or `MiddleColumn6` for 6/12 in 12 separated grid (all names hypothetical). Then once you come back to your component you can read through your jsx and at the first glance you know which tag does what and where is it placed in the layout, without even launching your project!
> 
> I hope it all makes sense :)

Thank you so much for your answer! 
We are kind of new to styled components, Hopefully, will get used to it soon :) 
Is anything that we should fix? if not, can you approve?